src/utils/Utils.ts

```diff
@@ -86,6 +87,50 @@ export default class Utils {
     });
   }
 
+  public static optimizationAlgorithmsToList(algorithms: OptimizationAlgorithm[], onClick?: clickHandler) {
+    return algorithms.map((algorithm) => {
+      return {
+        id: algorithm.id || algorithm.name,
+        firstValue: algorithm.name,
+        secondValue:
+          `( ${algorithm.inputs.join(', ')} ) => ${algorithm.outputs}`,
+        thirdValue: '',
+        link: onClick ? {
+          onClick: () => { onClick(algorithm.id || algorithm.name); },
+        } : {
+          link: { name: 'algorithm', params: { id: algorithm.id }},
```

Comment:
why do we need a link here? we are already on this page.
I know, the resource type view has also the link to itself, but i would ask the same question there.

Reply:
The thing is, this util function builds the single algorithm view as well as the list.